framework/src/node/bft/api.ts

```diff
@@ -214,6 +204,9 @@ export class BFTAPI extends BaseAPI {
 
 		const nextActiveValidators: BFTVotesActiveValidatorsVoteInfo[] = [];
 		for (const validator of validators) {
+			if (validator.bftWeight === BigInt(0)) {
```

Comment:
ignoring 0 bft weight for active validator

Reply:
In the BFT LIP, validators with BFT weight 0 are not accepted in `setBFTParameters` and the API fails. If I understand it correctly here, now it is possible to set BFT weight to 0 and the validator is just skipped over. So the state information in the end would be the same. Is that correct?

Reply:
State information for the `bftVotes` will remain the same. However, BFT params will maintain 0 vote weight validators